app/mm-redux/utils/channel_utils.ts

```diff
@@ -736,3 +736,14 @@ export function filterChannelsMatchingTerm(channels: Array<Channel>, term: strin
             displayName.startsWith(lowercasedTerm);
     });
 }
+
+export function removeItem<T>(array: T[], item: T) {
```

Comment:
this function does not really seem related to channels but more of a generic way to remove an element from an array.. does it belong in this file ?

Reply:
I ported it over from the webapp; it's used inside `orderByTeam` where the comment is:

`removeItem only modifies the array if it contains the category ID, so other teams' state won't be modified` 👀